### DISCUSSION TITLE
Type mismatch when creating `DefaultServiceManager` with Kotlin

### DISCUSSION THREAD

**Discussion by Greatwallcorner** [jupnp/jupnp#223] [Q&A]
I got 
![图片](https://github.com/jupnp/jupnp/assets/49832311/7a192f37-147f-48cb-9998-71d17a055ea4)
when I try to create a upnp service with cling's guide. Anyone got this? Expect your reply.
![图片](https://github.com/jupnp/jupnp/assets/49832311/c0abbb20-bddf-4a7b-8476-dfeac50a3c5d)
```
 implementation("org.jupnp:org.jupnp:2.7.1")
 implementation("org.jupnp:org.jupnp.support:2.7.1")
```

**Comment by wborn** [comment]
I don't know much about Kotlin but maybe you can give the `s` variable a type instead of using `val` or cast it to the right type when using it in the `DefaultManagerService` constructor?
